src/renderer/components/ft-playlist-add-video-button/ft-playlist-add-video-button.js

```diff
@@ -0,0 +1,125 @@
+import { defineComponent } from 'vue'
+import FtIconButton from '../ft-icon-button/ft-icon-button.vue'
+import { mapActions } from 'vuex'
+
+export default defineComponent({
+  name: 'FtPlaylistAddVideoButton',
+  components: {
+    'ft-icon-button': FtIconButton,
+  },
+  props: {
+    videoId: {
+      type: String,
+      required: true
+    },
+    videoTitle: {
+      type: String,
+      required: true
+    },
+    channelId: {
+      type: String,
+      required: true
+    },
+    channelName: {
+      type: String,
+      required: true
+    },
+    viewCount: {
+      type: Number,
+      required: false,
+      default: null,
+    },
+    lengthSeconds: {
+      type: Number,
+      required: true
+    },
+
+    padding: {
+      type: Number,
+      default: 10
+    },
+    size: {
+      type: Number,
+      default: 20
+    },
+
+    showVideoAddedToPlaylistCount: {
+      type: Boolean,
+      default: true,
+    },
+  },
+  computed: {
+    allPlaylists: function () {
+      return this.$store.getters.getAllPlaylists
+    },
+
+    title() {
+      const initialText = this.$t('User Playlists.Add to Playlist')
+      if (this.videoAddedToPlaylistCountText == null) { return initialText }
+
+      return `${this.$t('User Playlists.Add to Playlist')} (${this.videoAddedToPlaylistCountText})`
+    },
+
+    videoAddedToPlaylistCount() {
+      if (!this.showVideoAddedToPlaylistCount) { return 0 }
+
+      let count = 0
+
+      this.allPlaylists.forEach((playlist) => {
+        const videoAlreadyAdded = playlist.videos.some((v) => {
+          return v.videoId === this.videoId
+        })
+        if (videoAlreadyAdded) { count += 1 }
+      })
```

Comment:
Same optimisation as in https://github.com/FreeTubeApp/FreeTube/pull/5170
```suggestion
      const videoId = this.videoId
      let count = 0

      this.allPlaylists.forEach((playlist) => {
        const videoAlreadyAdded = playlist.videos.some((v) => {
          return v.videoId === videoId
        })
        if (videoAlreadyAdded) { count += 1 }
      })
```